drivers/SmartThings/zigbee-contact/src/multi-sensor/multi_utils.lua

```diff
@@ -138,7 +138,7 @@ multi_utils.convert_to_signedInt16 = function(byte1, byte2)
   local finalValue
   local swapped = (byte2 << 8) | byte1
   local sign_mask = 0x8000
-  local int16mask = 0xFF
+  local int16mask = 0xFFFF
```

Comment:
wouldn't this now be an int32 mask?

Reply:
0xFFFF is 16-bits. See https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1361 for more details about this change